ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/CaseMapper.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCase;
+import uk.gov.hmcts.cmc.ccd.domain.CCDDocument;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+import java.net.URI;
+import java.util.Arrays;
+
+import static uk.gov.hmcts.cmc.ccd.domain.CCDYesNoOption.NO;
+import static uk.gov.hmcts.cmc.ccd.domain.CCDYesNoOption.YES;
+
+@Component
+public class CaseMapper {
+
+    private final ClaimMapper claimMapper;
+
+    public CaseMapper(ClaimMapper claimMapper) {
+        this.claimMapper = claimMapper;
+    }
+
+    public CCDCase to(Claim claim) {
+        final CCDCase.CCDCaseBuilder builder = CCDCase.builder();
+
+        if (claim.getLetterHolderId() != null) {
+            builder.letterHolderId(claim.getLetterHolderId());
+        }
+
+        if (claim.getDefendantId() != null) {
```

Comment:
we dont need these if. just use builder.defendantId.. if claim.getDefendantId is null, it will get assigned as null.

Reply:
defendant Id is removed from case